File: 2496.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
  int a, b, c;
  char n[28];
  string  q ("ABCDEFGHIJKLMNOPQRSTUVWXYZ");
  cin >> a;
  while(a--){
    cin >> b;
    scanf("%s",n);
    c=0;
    for(int i=0;i<b;i++){
      if(n[i]!=q[i]){
	c++;}
    }
  
    if(c>2){
      cout << "There aren't the chance.\n";
    }
    else{
      cout << "There are the chance.\n";
    }
  }
}

File: 2496.py
n = int(input())
s = list("ABCDEFGHIJKLMNOPQRSTUVWXYZ")
while(n>0):    
    num = int(input())
    nome = list(input())
    k = 0
    i = 0
    j = 0
    a = 0
    for k in range(num):
        if(s[i] == nome[i]):
            i+=1
        else:
            j = i
            while(s[i]!= nome[j]):
                j+=1
            temp = nome[j]
            nome[j] = nome[i]
            nome[i] = temp
            a += 1
            
    if(a<=1):
        print("There are the chance.")
    else:
        print("There aren't the chance.")
        
    n-=1
    
